data/json/monster_factions.json

```diff
@@ -123,5 +123,20 @@
         "type" : "MONSTER_FACTION",
         "name" : "spider_cellar",
         "base_faction" : "spider"
+    },{
+        "type" : "MONSTER_FACTION",
+        "name" : "mutant",
+        "neutral" : ["vermin"],
+        "by_mood" : ["small_animal", "blob", "utility_bot", "mutant_omnivore"]
```

Comment:
No need to name all those factions here, unless base faction overrides them.

Default faction is "by_mood" to everything except vermin and itself.